src/main/java/attendance/dto/AttendResult.java

```diff
@@ -0,0 +1,10 @@
+package attendance.dto;
+
+import attendance.model.AttendanceType;
+import java.time.LocalDateTime;
+
+public record AttendResult(
```

Comment:
dto 클래스는 클래스명에 dto임을 표현해주세요~
```suggestion
public record AttendResultDto(
```

Reply:
더 명확한 방법인 것 같습니다! 감사합니다.